demo/src/assets/json/versions.json

```diff
@@ -1 +1 @@
-[]
+[];
```

Comment:
semicolon in json file?
I suppose it should break parsing `JSON.parse('[];')`

Reply:
Sure! I fixed it.